KalturaMediaServer/src/main/java/com/kaltura/media/server/managers/KalturaLiveManager.java

```diff
@@ -127,8 +127,8 @@ public boolean removeReferrer(ILiveEntryReferrer obj) {
 		}
 
 		public LiveEntryCache(KalturaLiveEntry liveEntry, KalturaMediaServerIndex serverIndex, String applicationName) {
-			this(liveEntry);
-			register(serverIndex, applicationName);
+			// Check if this constructor is even relevant? Might be possible to delete?
+			register();
```

Comment:
there is a difference between the target branch and your branch, you should resolve the conflics